numpy/core/getlimits.py

```diff
@@ -359,6 +358,18 @@ class finfo:
     impacts import times.  These objects are cached, so calling ``finfo()``
     repeatedly inside your functions is not a problem.
 
+    Note that `tiny` is not actually the smallest positive representable
+    value in a NumPy floating point type. As in the IEEE-754 standard[1]_,
+    NumPy floating point types make use of subnormal numbers to fill the
+    gap between 0 and `tiny`. However, subnormal numbers may have a 
+    significantly reduced precision[2]_.
```

Comment:
```suggestion
    Note that ``tiny`` is not actually the smallest positive representable
    value in a NumPy floating point type. As in the IEEE-754 standard [1]_,
    NumPy floating point types make use of subnormal numbers to fill the
    gap between 0 and ``tiny``. However, subnormal numbers may have
    significantly reduced precision [2]_.
```
 * tiny should be an inline literal rather than emph
 * I think there needs to be a space before citations for numpydoc
 * grammatical nit

Reply:
All fine with me! Many thanks!